app/src/main/kotlin/io/intrepid/skotlinton/screens/example2/Example2Fragment.kt

```diff
@@ -11,9 +11,9 @@ import io.intrepid.skotlinton.base.PresenterConfiguration
 class Example2Fragment : BaseFragment<Example2Contract.Presenter>(), Example2Contract.View {
 
     @BindView(R.id.example2_current_ip)
-    internal lateinit var currentIpView: TextView
+    private lateinit var currentIpView: TextView
     @BindView(R.id.example2_previous_ip)
-    internal lateinit var previousIpView: TextView
+    private lateinit var previousIpView: TextView
```

Comment:
Does this work? Pretty sure these need to be `protected`. See also discussion [here](https://github.com/IntrepidPursuits/skotlinton-android/pull/21#discussion_r163985241) from #21 (which also hits some of the other changes in here)

Reply:
Yeah looks like this breaks the build since butterknife can't read it

Reply:
I'll just keep this PR to the dependencies then since the other PR covers the lint warning cleanup